src/TestSuite/TestCase.php

```diff
@@ -241,52 +214,56 @@ protected function setUp(): void
     protected function tearDown(): void
     {
         parent::tearDown();
+        $this->teardownFixtures();
+
         if ($this->_configure) {
             Configure::clear();
             Configure::write($this->_configure);
         }
         $this->getTableLocator()->clear();
         $this->_configure = [];
         $this->_tableLocator = null;
-
-        if ($this->fixtureManager) {
-            $this->fixtureManager->teardownTest($this);
-            $this->fixtureManager = null;
-        }
     }
 
     /**
-     * Chooses which fixtures to load for a given test
-     *
-     * Each parameter is a model name that corresponds to a fixture, i.e. 'Posts', 'Authors', etc.
-     * Passing no parameters will cause all fixtures on the test case to load.
+     * Initialized and loads any use fixtures.
      *
      * @return void
-     * @see \Cake\TestSuite\TestCase::$autoFixtures
-     * @throws \RuntimeException when no fixture manager is available.
      */
-    public function loadFixtures(): void
+    protected function setupFixtures(): void
     {
-        if ($this->autoFixtures) {
-            throw new RuntimeException('Cannot use `loadFixtures()` with `$autoFixtures` enabled.');
-        }
-        if ($this->fixtureManager === null) {
-            throw new RuntimeException('No fixture manager to load the test fixture');
+        $fixtureNames = $this->getFixtures();
+        if (empty($fixtureNames)) {
+            return;
         }
 
-        $args = func_get_args();
-        foreach ($args as $class) {
-            $this->fixtureManager->loadSingle($class, null);
-        }
+        $this->fixtureStrategy = $this->getFixtureStrategy();
```

Comment:
This uses a getter instead of a class config, but we need to cache off the strategy for the duration of the test.